Makefile

```diff
@@ -6,6 +6,15 @@
 # Ensure go-extpoints and go-import-subtree are available for go generate
 export PATH := $(GOPATH)/bin:$(PATH)
 
+dev:
+	go get github.com/axw/gocov/gocov
+	go get github.com/golang/lint/golint
+	go get github.com/mattn/goveralls
+	go get github.com/pierrre/gotestcover
+	go get github.com/stretchr/testify/assert
```

Comment:
Works for me. @selenamarie can you try running this script? It builds a fresh version, with lines 12 and 14 deleted.

```bash
#!/bin/bash -xv

cd "$(dirname "${0}")"
rm -rf tmpgopath
mkdir tmpgopath
export GOPATH=$(pwd)/tmpgopath
go get github.com/taskcluster/taskcluster-worker
cd tmpgopath/src/github.com/taskcluster/taskcluster-worker
git remote add selena git@github.com:selenamarie/taskcluster-worker.git
git fetch selena
git checkout readme-update
cat Makefile | sed -e 12d -e 14d > m
mv m Makefile
git merge -m "pulling in latest changes" origin/master # needed for compatibility with latest taskcluster-client-go
make dev test
echo "'make dev test' exit code was: $?"
cd ../../../../..
rm -rf tmpgopath
```

Let me know if you get:
```
'make dev test' exit code was: 0
```